docs/docs/rest-api/public/api/v2/types/deploymentPlan.raml

```diff
@@ -15,7 +15,7 @@ types:
         type: string
         description: The UUID of the deployment plan
       steps:
-        type: deploymentSteps.DeploymentSteps
+        type: deploymentSteps.DeploymentStep[]
```

Comment:
This is a breaking API change, isn't it? We now have:
```
"steps": [
   { ...step... },
   { ...step...},
]
```
intead of:
```
"steps": {
  "steps": [
    { ...step...},
    { ...step...}
  ]
}
```

Or do I read that wrong?

Reply:
No. I had to look this up. `deploymentSteps.DeploymentStep` has no `steps` field but `deploymentSteps.DeploymentStepS` has.

Reply:
I see the same thing as @ANeumann82 ; the RAML was actually wrong but we weren't using the RAML to serialize the deployment plan (https://github.com/mesosphere/marathon/pull/7081/files#diff-df71fde3a131a175b64c40a758c0423dL203)